flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/DefaultSlotPoolServiceSchedulerFactoryTest.java

```diff
@@ -96,4 +148,16 @@ private static Stream<Arguments> testGetRequestSlotMatchingStrategy() {
                         JobType.STREAMING,
                         PreferredAllocationRequestSlotMatchingStrategy.INSTANCE));
     }
+
+    private String saveAdaptiveSchedulerTestPropertyValue() {
+        return (String) System.getProperties().remove("flink.tests.enable-adaptive-scheduler");
```

Comment:
I would not remove it here but explicitly remove it in `testFallBackSchedulerWithoutAdaptiveSchedulerTestProperty`